src/averageAges.js

```diff
@@ -15,31 +15,53 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let peopleCentury = [...people];
+
+  if (century !== null && typeof century === 'number') {
```

Comment:
This check can be simplified
```suggestion
  if (!century) {
```